src/screens/CreatePost/elements/SheetExpiredPost.js

```diff
@@ -4,54 +4,53 @@ import {StyleSheet, Text, View} from 'react-native';
 import RBSheet from 'react-native-raw-bottom-sheet';
 
 import FlatListItem from '../../../components/FlatListItem';
+import {colors} from '../../../utils/colors';
 import {fonts} from '../../../utils/fonts';
-import {COLORS} from '../../../utils/theme';
 
-const SheetExpiredPost = ({refExpired, data, select, onSelect, isAnonimity = false}) => (
-  <RBSheet
-    ref={refExpired}
-    closeOnDragDown={true}
-    closeOnPressMask={true}
-    customStyles={{
-      container: styles.containerSheet,
-      draggableIcon: styles.draggableIcon
-    }}>
-    <View style={styles.container}>
-      <Text style={styles.header}>Post expiration after</Text>
-      {data.map((value, index) => (
-        <FlatListItem
-          key={index}
-          value={value.label}
-          index={index}
-          select={select}
-          onSelect={onSelect}
-          isAnonimity={isAnonimity}
-        />
-      ))}
-    </View>
-  </RBSheet>
-);
+const SheetExpiredPost = ({refExpired, data, select, onSelect}) => (
+    <RBSheet
+      ref={refExpired}
+      closeOnDragDown={true}
+      closeOnPressMask={true}
+      customStyles={{
+        container: styles.containerSheet,
+        draggableIcon: styles.draggableIcon,
+      }}>
+      <View style={styles.container}>
+        <Text style={styles.header}>Post expiration after</Text>
+        {data.map((value, index) => (
+          <FlatListItem
+            key={index}
+            value={value.label}
+            index={index}
+            select={select}
+            onSelect={onSelect}
+          />
+        ))}
+      </View>
+    </RBSheet>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `SheetExpiredPost` component has been updated to remove the `isAnonimity` prop. This is a significant change and could potentially break functionality if other parts of the codebase are still passing this prop and expecting it to be used. Ensure that this prop is no longer necessary or that all instances where it was being passed have been updated.
```diff
- const SheetExpiredPost = ({refExpired, data, select, onSelect, isAnonimity = false}) => (
+ const SheetExpiredPost = ({refExpired, data, select, onSelect}) => (
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->